config/initializers/rack_attack.rb

```diff
@@ -0,0 +1,20 @@
+if Rails.env.development? || Rails.env.test?
+  Rack::Attack.cache.store = ActiveSupport::Cache::MemoryStore.new
+  Rack::Attack.enabled = false
+else
+  redis_url = PaasConfigurationService.new.redis_uris[:"dluhc-core-#{Redis.env}-redis-rate-limit"]
```

Comment:
@smeny Do we need to set up any of the env vars etc before we deploy this? Also, is the Redis instance on staging now?

Reply:
* No, because when the redis instance is bound to our CORE app, it adds the section in VCAP_SERVICES (which we read from). Note that we should update the deployment manifest to ensure this link is enforced.
* Yes, the redis instance is available on staging.